docs/api/batch.md

```diff
@@ -6,8 +6,8 @@ The Batch API is used to request the ability to transfer LFS objects with the
 LFS server. The Batch URL is built by adding `/objects/batch` to the LFS server
 URL.
 
-Git remote: https://git-server.com/foo/bar  
-LFS server: https://git-server.com/foo/bar.git/info/lfs  
+Git remote: https://git-server.com/foo/bar
+LFS server: https://git-server.com/foo/bar.git/info/lfs
```

Comment:
Gah, small markdown issue. The 2 trailing spaces are a line break. I suppose this could either be a markdown list, or lines could end with an explicit `<br>`.